docs/data/trigger-sync.md

```diff
@@ -0,0 +1,270 @@
+3. Click **Add module**, then enter a name for your sensor and click **Create**.
+   The sensor will return true and enable sync when in a specified time frame.
+4. To configure your timeframe, go to the new component panel and copy and paste the following attribute template into your sensor’s **Attributes** box:
+
+   {{< tabs >}}
+   {{% tab name="Config builder" %}}
+
+```json
+{
+  "start": "HH:MM:SS",
+  "end": "HH:MM:SS",
+  "zone": "<TIMEZONE>"
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json
+{
+  "name": "<SENSORNAME>",
+  "model": "naomi:sync-at-time:timesyncsensor",
+  "type": "sensor",
+  "namespace": "rdk",
+  "attributes": {
+    "start": "HH:MM:SS",
+    "end": "HH:MM:SS",
+    "zone": "<TIMEZONE>"
+  },
+  "depends_on": []
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json
+{
+  "name": "timesensor",
+  "model": "naomi:sync-at-time:timesyncsensor",
+  "type": "sensor",
+  "namespace": "rdk",
+  "attributes": {
+    "start": "18:29:00",
+    "end": "18:30:00",
+    "zone": "CET"
+  },
+  "depends_on": []
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for the `naomi:sync-at-time:timesyncsensor` sensor:
+
+<!-- prettier-ignore -->
+| Name    | Type   | Inclusion    | Description |
+| ------- | ------ | ------------ | ----------- |
+| `start` | string | **Required** | The start time for the time frame during which you want to sync, example: `"14:10:00"`.  |
+| `end`   | string | **Required** | The start time for the time frame during which you want to sync, example: `"15:35:00"`. |
+| `zone`  | string | **Required** | The time zone for the `start` and `end` time, for example: `"CET"`. |
+
+In the next step you will configure the data manager to take the sensor into account when syncing.
+
+### Configure data manager to sync based on sensor
+
+On your machine's **Config** tab, switch to **JSON** mode and add a `selective_syncer_name` with the name for the sensor you configured:
+
+{{< tabs >}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers" data-line="6"}
+{
+  "attributes": {
+    "additional_sync_paths": [],
+    "capture_dir": "",
+    "capture_disabled": false,
+    "selective_syncer_name": "<SENSOR-NAME>",
+    "sync_disabled": false,
+    "sync_interval_mins": 0.1,
+    "tags": []
+  },
+  "name": "Data-Management-Service",
+  "type": "data_manager"
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers" data-line="6"}
+{
+  "attributes": {
+    "additional_sync_paths": [],
+    "capture_dir": "",
+    "capture_disabled": false,
+    "selective_syncer_name": "timesensor",
+    "sync_disabled": false,
+    "sync_interval_mins": 0.1,
+    "tags": []
+  },
+  "name": "Data-Management-Service",
+  "type": "data_manager"
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% expand "Click to view a full configuration example" %}}
+
+```json
```

Comment:
[opt] Consider adding line highlighting to the whole cloud sync section

Reply:
```suggestion
```json {class="line-numbers linkable-line-numbers" data-line="7,12"}
```